pages.zh/linux/cp.md

```diff
@@ -0,0 +1,32 @@
+# cp
+
+> 复制文件和目录。
+> 更多信息：<https://www.gnu.org/software/coreutils/cp>.
+
+- 复制一个文件到另外一个地方：
+
+`cp {{path/to/source_file.ext}} {{path/to/target_file.ext}}`
```

Comment:
Please translate the `path/to/...`  for get this easier for non-English speakers, in all pages.